docs/action-debugging.md

```diff
@@ -1,9 +1,23 @@
 # Debugging
-If the build logs do not provide enough detail on why a build may be failing, some other options exist to assist with troubleshooting.
+If the job logs do not provide enough detail on why a job may be failing, some other options exist to assist with troubleshooting.
+
+## Step Debug Logs
+This is the primary way for customer to debug their job failure that caused by certain step failed.
```

Comment:
I'd suggest changing to:
This is the primary way for customers to debug job failures caused by step failures.